### PR TITLE
fix(logging): cannot migrate error message

### LINKED ISSUE #1198: `Option` (implementation detail) leaks into log output
### Bug description

I don't like that the `Option` leaks into the log ... we should call `.unwrap_or(0)` on it.

image

### PR DESCRIPTION
**Pull request by Alex6323**
## Linked Issues

* Closes #1198

### CODE REVIEW

**Review by Alexandcoats — CHANGES_REQUESTED** [review]
Nope, this cannot work. Migration 0 is a valid version, so printing this would be wrong. I personally don't see the issue with printing an option value (we do it in many places, I am certain), but if you really want to remove that then you'll have to print two different messages based on the value. Or you could just unwrap it, since `None` should always lead to a valid migration.

**Comment by Alex6323** [comment]
yeah, I got confused and took the `None` for version `0`, but it's actually `Some(0)` what we're getting, so this cannot work.

File: src/bin/inx-chronicle/migrations/mod.rs [hunk 1]
```diff
@@ -118,9 +118,10 @@ pub async fn migrate(db: &MongoDb) -> eyre::Result<()> {
                 migration.migrate(db).await?;
             }
             None => {
+                // Unwrap: it's safe to unwrap `last_migration` in this case.
```
Comment (Alexandcoats) [inline src/bin/inx-chronicle/migrations/mod.rs:121]: Perhaps explain why?

Comment (Alex6323) [reply src/bin/inx-chronicle/migrations/mod.rs:121]: sure

**Review by Alexandcoats — APPROVED** [review]

**Pull request merged**